keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -133,6 +133,14 @@ def augment_image(self, image, transformation=None, **kwargs):
     def augment_label(self, label, transformation=None, **kwargs):
         return label
 
+    def augment_bounding_boxes(self, bounding_boxes, **kwargs):
+        # TODO: implements
```

Comment:
Let's remove these, we don't want TODO: implements.  This introduces a silent failure in RandomShear.

It is an intentional design choice that layers that do not implement the method error out.

Reply:
@atuleu any update?  I'd love to get this merged